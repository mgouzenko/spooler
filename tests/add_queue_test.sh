addqueue tests/test_dir/foo
addqueue tests/test_dir/world
addqueue tests/test_dir/cat tests/test_dir/dog
cat /var/spool/printer/world_1
