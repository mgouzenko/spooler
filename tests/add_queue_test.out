tests/test_dir/foo: X Permission denied
tests/test_dir/world: Y 1
tests/test_dir/cat: Y 2
tests/test_dir/dog: X Permission denied
hello
